CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+All contributions are welcome to this project!
+
+The general process for contributing code is as follows:
+
+1. Open an issue describing the desired feature, bug fix, or improvement you'd like to see before making any changes.
+2. Assign the ticket to yourself, and/or leave a comment on the issue that you would like to work on it.
```

Comment:
The contributor should wait for confirmation from the project before starting work, as this one isn't rigorously maintained.